src/pages/index.astro

```diff
@@ -7,5 +7,5 @@ const jsons : Lingo[] = await Astro.glob("../../lingos/*.json");
 ---
 
 <Layout title="TechLingo.fyi" description="TechLingo.fyi is a free resource for people who want to learn about the jargon and acronyms used in the tech industry." path="">
-    <LingoCards lingos={jsons} />
+    <LingoCards lingos={jsons} client:load />
```

Comment:
it's the first time I work with astro, I found that astro strips out all client-side JavaScript automatically, to create dynamic components is necessary the `client:load` directive

Reply:
Yeah, I am not that familiar with Astro either 🤷‍♂️